build.gradle.kts

```diff
@@ -243,15 +296,20 @@ tasks.register("createApp") {
   }
 }
 
+/*
+ * Task: createDmg
+ *
+ * Creates macOS DMG package file.
+ */
 tasks.register("createDmg") {
   group = "build"
-  description = "Makes the Mac dmg package"
+  description = "Makes the Mac DMG package file."
```

Comment:
`DMG package` is a kind of file, so the "file" at the end of the line is redundant.